newIDE/app/src/ResourcesList/ResourcePreview/ImagePreview.js

```diff
@@ -77,6 +79,9 @@ type Props = {|
   onSize?: (number, number) => void,
   hideCheckeredBackground?: boolean,
   hideControls?: boolean,
+  isPrivate?: boolean,
```

Comment:
Same comment about private being a very overloaded term, let's qualify this prop as much as we can.